assets/dev/scss/direction/app-base.scss

```diff
@@ -0,0 +1,14 @@
+$direction: ltr;
+
+@import "../../../../assets/dev/scss/helpers/variables";
+
+@import "../../../../core/app/assets/styles/_functions.scss";
+@import "../../../../core/app/assets/styles/tokens.scss";
+@import "../../../../core/app/assets/styles/mixins.scss";
+@import "../../../../core/app/assets/js/ui/grid/grid";
+
+@import "../../../../core/app/assets/styles/base/utilities";
+@import "../../../../core/app/assets/styles/base/animations";
+@import "../../../../core/app/assets/js/ui/molecules/buttons";
+@import "../../../../core/app/assets/js/ui/modal/modal";
```

Comment:
Extract to a file that imported here and in the full app file.

Reply:
Handled.